### DISCUSSION TITLE
v0.2.0 release checklist

### DISCUSSION THREAD

**Discussion by TimoGlastra** [openwallet-foundation/credo-ts#622] [Announcements]
A lot of features are being worked on. Here a checklist of what we want to merge before making the 0.2.0 release. Suggestions on what should be included welcome!

- [x] https://github.com/hyperledger/aries-framework-javascript/issues/345 (@jakubkoci)
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/344 (@jakubkoci)
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/352 (@NB-MikeRichardson)
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/493 (@JamesKEbert, @TheTreek)
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/349 (@JamesKEbert, @TheTreek)
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/350 (@JamesKEbert, @TheTreek)
- [x] #305 (@TimoGlastra)
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/738 (@TimoGlastra)
	- PR: https://github.com/hyperledger/aries-framework-javascript/pull/844
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/737
	- PR: https://github.com/hyperledger/aries-framework-javascript/pull/841

Will be moved to 0.3.0 release:
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/755
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/353 (@karimStekelenburg, @NB-PrasadKatkar)
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/494 (@karimStekelenburg, @NB-PrasadKatkar)

Milestone: https://github.com/hyperledger/aries-framework-javascript/milestone/4

**Comment by TimoGlastra** [comment]
@blu3beri @JamesKEbert @jakubkoci @karimStekelenburg Is there anything missing here?

**Comment by JamesKEbert** [comment]
This looks good IMO 👍

**Comment by TimoGlastra** [comment]
Locked conversation as 0.2.0 has been released
